qa/workunits/rados/test_rados_tool.sh

```diff
@@ -591,5 +567,10 @@ test_cleanup
 test_append
 test_put
 
+# clean up environment, delete pool
+$CEPH_TOOL osd pool delete $POOL $POOL --yes-i-really-really-mean-it
+$CEPH_TOOL osd pool delete $POOL_EC $POOL_EC --yes-i-really-really-mean-it
+$CEPH_TOOL osd pool delete $POOL_CP_TARGE $POOL_CP_TARGET --yes-i-really-really-mean-it
```

Comment:
done, thanks.

by the way, it seems that "ceph osd pool delete" will always returns 0 whether it's successfully or not.